### ISSUE TITLE
Option for borders to render inside the window instead as fullscreen windows and multimonitor setup won't be as visible.

### ISSUE THREAD

**Issue by momomo** [FelixKratz/JankyBorders#126]
Option for borders to render inside the window instead as fullscreen windows and multimonitor setup won't be as visible, plus it also take up more space than the inner variant. 

Honer has support for this so it should be a matter of changing some property, like the size of the area relative to center to be slightly smaller.

**Comment by mooktakim** [comment]
I posted this also: https://github.com/FelixKratz/JankyBorders/issues/127

This feature would be great 👍

**Comment by momomo** [comment]
Yes, but I am still using Honer because of this. This is an better app but until they add this feature Honer still beats despite it crashing all day long.

**Comment by jakeonrails** [comment]
I came here for a Honer alternative and need that same functionality - otherwise I can't see the borders ever since I'm almost always maximized.

**Comment by qns7** [comment]
I would love this option too.
